src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1046,6 +1046,7 @@
   "com.minecolonies.coremod.gui.workerhuts.stonesmelter": "Stone Smeltery",
 
   "com.minecolonies.coremod.ai.wrongfood": "The food I have is too raw to eat!",
+  "com.minecolonies.coremod.ai.betterfood": "The food I have is not good enough for me to eat!",
```

Comment:
@ravenbuilder934 suggestions?

Reply:
could we do separate ones for kids and adults? like the kid one would be "Ew! I don't want to eat *that*!" and the adult would be "Can I have better food please?" or sth?

Reply:
yes, that's possible